repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/digital-transformation/newscategories.yaml

```diff
@@ -0,0 +1,144 @@
+    /selection:listitem[13]:
+      jcr:primaryType: selection:listitem
+      selection:key: professional-networks
+      selection:label: Professional networks
+    /selection:listitem[14]:
+      jcr:primaryType: selection:listitem
+      selection:key: consultations-and-surveys
+      selection:label: Consultations and surveys
+    /selection:listitem[15]:
+      jcr:primaryType: selection:listitem
+      selection:key: digital-roles-and-career-pathways
+      selection:label: Digital roles and career pathways
+    /selection:listitem[16]:
+      jcr:primaryType: selection:listitem
+      selection:key: occupational-frameworks
+      selection:label: Occupational frameworks
+    /selection:listitem[17]:
+      jcr:primaryType: selection:listitem
+      selection:key: digital-capability-frameworks
+      selection:label: Digital capability frameworks
+    /selection:listitem[18]:
+      jcr:primaryType: selection:listitem
+      selection:key: strategic-workforce-planning-and-modelling
+      selection:label: Strategic workforce planning and modelling
+    /selection:listitem[19]:
+      jcr:primaryType: selection:listitem
+      selection:key: industry
+      selection:label: Industry
+    /selection:listitem[20]:
+      jcr:primaryType: selection:listitem
+      selection:key: artificial-intelligence-and-robotics
+      selection:label: Artificial intelligence and robotics
+    /selection:listitem[21]:
+      jcr:primaryType: selection:listitem
+      selection:key: policy-and-strategy
+      selection:label: Policy and strategy
+    /selection:listitem[22]:
+      jcr:primaryType: selection:listitem
+      selection:key: nursing-and-midwifery
+      selection:label: Nursing and midwifery
+    /selection:listitem[23]:
+      jcr:primaryType: selection:listitem
+      selection:key: finance-and-procurement
+      selection:label: Finance and procurement
+    /selection:listitem[24]:
+      jcr:primaryType: selection:listitem
+      selection:key: allied-health-professions-(ahps)
+      selection:label: Allied Health Professions (AHPs)
+    /selection:listitem[25]:
+      jcr:primaryType: selection:listitem
+      selection:key: psychological-professions
+      selection:label: Psychological professions
+    /selection:listitem[26]:
+      jcr:primaryType: selection:listitem
+      selection:key: clinical-informatics
+      selection:label: Clinical informatics
+    /selection:listitem[27]:
+      jcr:primaryType: selection:listitem
+      selection:key: dental
+      selection:label: Dental
+    /selection:listitem[28]:
+      jcr:primaryType: selection:listitem
+      selection:key: estates-and-facilities
+      selection:label: Estates and facilities
+    /selection:listitem[29]:
+      jcr:primaryType: selection:listitem
+      selection:key: healthcare-science
+      selection:label: Healthcare science
+    /selection:listitem[30]:
+      jcr:primaryType: selection:listitem
+      selection:key: human-resources-/-od
```

Comment:
Could we remove `/` from this key ?

Reply:
Done. Also removed one entry that had brackets.

Reply:
Sure, thanks 👍